keps/sig-api-machinery/20180415-crds-to-ga.md

```diff
@@ -157,15 +162,24 @@ TODO: complete this list
 
 ### e2e tests for GA
 
-* Custom Resources should be readable and writable at all available versions (test for round trip-ability)
 * A Custom Resource watch should terminate if its CustomResourceDefinition is deleted or updated
 
 ### Scale Targets for GA
 
-* TODO quantify: Read/write latency of CRDs within X% of native Kubernetes types
-* TODO quantify: Latency degrades less than X% for up to 100k Custom Resources per CRD kind
-* TODO quantify: Webhook conversion QPS of a noop converter is within X% of QPS with no webhook
-* Coordinate with sig-scalability
+Scale targets here are both CRDs with and without a webhook converter.
```

Comment:
s/CRD/custom resource/, just to be clear that the targets are against the custom resources themselves, not the definitions